priv/static/common.css

```diff
@@ -0,0 +1,49 @@
+/* This file contains design tokens used in core Gleam projects */
+:root {
+  /* Greys */
+  --col-black: #000000;
+  --col-neutral-950: #151515;
+  --col-neutral-900: #1e1e1e;
+  --col-neutral-800: #2f2f2f;
+  --col-neutral-100: #f5f5f5;
+  --col-azure-900: #202431;
+  --col-azure-800: #292d3e;
+  --col-azure-600: #51597b;
+  --col-azure-500: #7e818b;
+  --col-sepia-450: #b1a894;
+  --col-sepia-400: #cac0a9;
+  --col-sepia-300: #e3d8be;
+  --col-sepia-100: #fffbe8;
+  --col-sepia-050: #fefefc;
+
+  /* Colors */
+  --col-magenta-800: #584355;
+  --col-magenta-300: #ffaff3;
+  --col-cyan-200: #a6f0fc;
+  --col-green-200: #c8ffa7;
+  --col-yellow-300: #ffd596;
+  --col-orange-400: #ff9d35;
+  --col-red-400: #ff6262;
```

Comment:
What happened to the colour names? What do these numbers mean?

Reply:
This file comes from this branding PR https://github.com/gleam-lang/branding/pull/12

The original color names weren't well received*, so instead I chose to do something similar to [what tailwind does](https://tailwindcss.com/docs/background-color) where the name specifies the general hue and intensity.

You can think of the numbers as of font weights, the larger numbers mean bolder font -> darker color.

*Hayleigh pointed out they don't explain well enough what the color looks like, and you didn't like how some names referenced countries/cultures

Generally these names are considered "private" to the `common.css` file and the "semantic" names that appear later in the file should be used

Reply:
> The original color names weren't well received

I don't know where you got this idea! We want to keep the named colours for sure.

Reply:
updated, is this what you had in mind?